### PR TITLE
Support nodes develop branch

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Made changes to support the latest CENNZnet node's develop branch.
 Updated the balance where it was big value to smaller value
Added a new test case to updated payee as any account
Updated metadata
One thing that I could not do is to update the interfaceRegistry to reflect the new type definition for RewardDestination..module augmentation used does not let me do that.. any pointers appreciated..

### CODE REVIEW

**Comment by KarishmaBothara** [comment]
> Made changes to support the latest CENNZnet node's develop branch.
> Updated the balance where it was big value to smaller value
> Added a new test case to updated payee as any account
> Updated metadata
> One thing that I could not do is to update the interfaceRegistry to reflect the new type definition for RewardDestination..module augmentation used does not let me do that.. any pointers appreciated..

Type definition for RewardDestination is now merged to support 'account' type.. thanks @holygits for this..

File: packages/api/test/e2e/cennzx.e2e.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ describe('CENNZX e2e queries/transactions', () => {
   describe('Queries()', () => {
 
     it("Deposit liquidity in CENNZ asset's pool", async done => {
-        const amount = 30000000000000;
+        const amount = 3000000;
```
Comment (holygits) [inline packages/api/test/e2e/cennzx.e2e.ts:28]: 3_000_000

File: packages/types/src/runtime/staking/types.spec.ts [hunk 2]
```diff
@@ -15,5 +15,17 @@ describe('RewardDestination', (): void => {
...
+
+    const destinationAccountAddress = '5FEe8Ht1ZTzNjQcvrxbLxnykA2EXfqN5LMog2gaNPus4tfZR';
+    const destinationAccount = createType(registry, 'RewardDestination', {account: destinationAccountAddress}, 2);
+    expect(destinationAccount.asAccount.toString()).toEqual(destinationAccountAddress);
+    expect(destinationAccount.isAccount).toBeTruthy();
+    const destinationAccountStr = createType(registry, 'RewardDestination', {account: destinationAccountAddress});
+    expect(destinationAccountStr.asAccount.toString()).toEqual(destinationAccountAddress);
+    expect(destinationAccountStr.isAccount).toBeTruthy();
+
+    const destinationPayee = new RewardDestination(registry, {account: destinationAccountAddress});
```
Comment (holygits) [inline packages/types/src/runtime/staking/types.spec.ts:27]: nit:
```suggestion
    const destinationPayee = new RewardDestination(registry, { account: destinationAccountAddress });
```

File: packages/types/src/runtime/staking/index.ts [hunk 3]
```diff
@@ -13,11 +13,30 @@
...
 // Specifies which account staking rewards should be paid too.
 // Note: Polkadot has an additional 'Staked' option which is not supported in CENNZnet
 // due to the fact that the reward currency is different from the staked currency.
-export class RewardDestination extends Enum.with({
-  Stash: 'Null',
-  Controller: 'Null',
-}) {}
+export class RewardDestination extends Enum {
+  constructor(registry: Registry, value: any) {
+    super(registry, {Stash: 'Null', Controller: 'Null', Account: 'AccountId'}, value);
```
Comment (holygits) [inline packages/types/src/runtime/staking/index.ts:24]: nit:
```suggestion
    super(registry, { Stash: 'Null', Controller: 'Null', Account: 'AccountId' }, value);
```

**Review by holygits — COMMENTED** [review]
changes lgtm!

**Review by holygits — APPROVED** [review]

File: packages/types/src/runtime/staking/types.spec.ts (hunk 2 above)
Comment (holygits) [reply packages/types/src/runtime/staking/types.spec.ts:27]: I though eslint would do this so not sure why it's different

Comment (KarishmaBothara) [reply packages/types/src/runtime/staking/types.spec.ts:27]: I will search which rule will do this and enable it explicitly in the eslintrc..

Comment (KarishmaBothara) [reply packages/types/src/runtime/staking/types.spec.ts:27]: "object-curly-spacing": [2, "always"] -- if I add this rule, I have 808 errors.. I will update this PR without the rule and modify at these two place and have another PR to fix those 808 errors.. there as option to fix it with code..will use that..

**Pull request merged**